lib/report_formatter/timeline.rb

```diff
@@ -52,6 +52,13 @@ def to_cid(id)
       ApplicationRecord.compress_id(id)
     end
 
+    def tl_href(col, rec, text, ems_cloud, ems_container)
+      tl_message = TimelineMessage.new(rec, text, ems_cloud, ems_container)
```

Comment:
we can reset them each time, but the code will be ugly :-( so leaving it as is for now.